.github/scripts/check_native_dispatch.py

```
@@ -0,0 +1,103 @@
+# *_NATIVE macros the probe should report (gating macros above + a few more)
+NATIVE_MACROS = sorted({m for lst in PRIORITY.values() for m, _ in lst})
+ASSERT_CAPS = ("native_crc32", "native_adler32")
+
+def main():
+    src, build, cc = sys.argv[1], sys.argv[2], sys.argv[3]
+    extra = sys.argv[4:]
+
+    log = ""
+    for c in (f"{build}/.cfg.log", *[f"{build}/{d}" for d in os.listdir(build)
+              if d == ".cfg.log"]):
+        try: log = open(c).read(); break
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)